connectors/src/lib/models/slack.ts

```diff
@@ -290,3 +290,46 @@ SlackChatBotMessage.init(
   }
 );
 ConnectorModel.hasOne(SlackChatBotMessage);
+
+export class SlackBotWhitelist extends Model<
```

Comment:
Can we use the same pattern as whitelistedDomain (JSON list) vs a new table)? So that the two are aligned?